knowledge-base/checkmarx-ts/checkmarx-cxflow-github-action/action-security.yml

```diff
@@ -0,0 +1,12 @@
+name: 'Checkmarx CxFlow Action'
+github-token:
+  action-input:
+    input: github_token
+    is-default: true
+  environment-variable-name: GITHUB_TOKEN
+  permissions:
+    pull-requests: write 
+    pull-requests-reason: to write Feedbacks to PR.
```

Comment:
Can you please change it to `to write feedback to PR`? Do not add a full stop, it is not there is other reasons. 
Same for issues-reason